core/templates/dev/head/pages/practice-session-page/practice-session-page.controller.ts

```diff
@@ -28,6 +28,10 @@ require(
   'components/question-directives/question-player/' +
   'question-player.directive.ts');
 
+require('objects/templates/MathLatexStringEditorDirective.ts');
+require('objects/templates/SanitizedUrlEditorDirective.ts');
+require('objects/templates/ListOfTabsEditorDirective.ts');
```

Comment:
Could you move these to a separate file (as in `extensions/rich_text_components`) and the require the file here and elsewhere?

Reply:
done, for these I created a separate TS file containing the requires since these controllers don't need the full set.